src/copy.js

```diff
@@ -22,15 +22,15 @@ document.addEventListener('copy', e => {
 });
 
 const updateLink = clipboardData => {
-  var modifiedLink = clipboardData.replace('twitter', 'vxtwitter');
+  var modifiedLink = clipboardData.replace(/twitter\.com|x\.com/g, 'vxtwitter.com');
```

Comment:
If you replace x with vxtwitter, it may unintentionally hit characters in usernames, etc., so we include .com as well.